bin/fixtures.js

```diff
@@ -1,28 +1,45 @@
+#!/usr/local/bin/node
+
 "use strict";
 
 const fs = require('fs');
 const path = require('path');
 
-const fixturePath = path.join(__dirname, "../public/js/test/fixtures");
-
-function saveFixture(name, text) {
-  console.log("starting saveFixture")
+function getFixtureFile(name) {
+  const fixturePath = path.join(__dirname, "../public/js/test/fixtures");
   const fixtureFile = path.join(fixturePath, name + ".json");
-
   if (!fs.existsSync(fixturePath)) {
     throw new Error("Could not find fixture " + name);
   }
 
-  console.log("writing file")
+  return fixtureFile;
+}
+
+function saveFixture(name, text) {
+  console.log("starting saveFixture")
+  const fixtureFile = getFixtureFile(name);
   fs.writeFileSync(fixtureFile, text)
   process.exit(0);
 }
 
+function appendFixture(name, text) {
+  console.log("starting saveFixture")
+  const fixtureFile = getFixtureFile(name);
+  fs.appendFileSync(fixtureFile, text)
+  process.exit(0);
+}
+
+console.log("starting fixtures.js")
 const shouldSave = process.argv.indexOf("--save") > -1;
+const shouldAppend = process.argv.indexOf("--append") > -1;
```

Comment:
wrote the story in a comment below, but the fixture files are big enough to necessitate chunking them up and appending to the file